kafka-ui-react-app/src/redux/reducers/topicMessages/__test__/fixtures.ts

```diff
@@ -15,6 +15,17 @@ export const topicMessagePayload: TopicMessage = {
     '{"host":"schemaregistry1","port":8085,"master_eligibility":true,"scheme":"http","version":1}',
 };
 
+export const topicMessagePayloadV2: TopicMessage = {
+  partition: 28,
+  offset: 88,
+  timestamp: new Date('2021-07-21T23:25:14.865Z'),
+  timestampType: TopicMessageTimestampTypeEnum.CREATE_TIME,
+  key: 'schema-registry',
+  headers: {},
+  content:
+    '{"host":"schemaregistry1","port":8085,"master_eligibility":true,"scheme":"http","version":1}',
+};
```

Comment:
As I can see, the above 7 lines are the exact same thing that appear in topicMessagePayload declared above. Can we reduce code duplication by storing the repeated information in a separate variable and using it by spreading at the end?

Reply:
Sure.